package.json

```diff
@@ -47,4 +47,4 @@
     "typescript": "^4.5.5"
   },
   "packageManager": "yarn@3.2.0"
-}
+}
```

Comment:
missing newline

Reply:
damnit lint! I need to figure out how to fix that